### ISSUE TITLE
Mandatory Fields

### ISSUE THREAD

**Issue by PhilMellor** [highrisedigital/wpbroadbean#16]
Plugin documentation at https://docs.google.com/spreadsheets/d/1Md4nMlSB6diMlTo49sru89UAqj4vqFzvFUObyMnphPY/edit#gid=0 indicates that several mandatory fields are required by the plugin - however the integration people at Broadbean found during testing that the plugin would not accept job listings without the additional field of "job_skill"

**Comment by wpmark** [comment]
Hi Phil, apologies for the issue you are having here. I think that documentation is in fact rather old and therefore a little unreliable. I need to remove that asap.

The plugin does use the field named `job_skills` in order to populate the Skills taxonomy for the job, however not including it should not cause the posting to fail. Could you give me any particular errors you are noticing etc.?

**Comment by PhilMellor** [comment]
Hi Mark - no need to apologise! You're doing an awesome job!

I'm afraid I can't give you a specific error message - the communication from BB was simply "it wouldn't accept new data without job_skills".

I've also made a code tweak today with regards to the application email needing to have the header "From:" or "Reply-To:" being set as the applicant's rather than the blog/admin email address. Once BB test it successfully I'll create new issue and inform you how I patched.
